docs/UserGuide.md

```diff
@@ -140,8 +141,9 @@ This is mostly sufficient for you to know how to use the command. Here are some
   since it does not conform to NUS course code format). Nevertheless, if you enter such a course code as input, the
   program will allow you to bypass the validation by adding `f/` to the end of the command.
 
-  Note that `f/` must come at the end, or immediately before a tag. For example, `add f/ n/Alice …` is allowed, but `add
-  n/ f/ Alice …` is not allowed.
+:bulb: Note that `f/` must come at the end, or immediately before a tag. <br>
+For example, `add f/ n/Alice e/alice@gmail.com r/STUDENT c/CS2103T` is allowed, 
+but `add n/ f/ Alice …` or `add n/Alice e/ f/alice@gmail.com`is not allowed.
```

Comment:
Missing space here.

Also please make formatting consistent --- all bulb must be inside blue box.

**Preview site with Jekyll before merge.**

Reply:
Where's the missing space here. I think it's correct right?